var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -69,6 +69,8 @@ class PyScipy(PythonPackage):
     patch('https://git.sagemath.org/sage.git/plain/build/pkgs/scipy/patches/extern_decls.patch?id=711fe05025795e44b84233e065d240859ccae5bd',
           sha256='5433f60831cb554101520a8f8871ac5a32c95f7a971ccd68b69049535b106780', when='@1.2:1.5.3')
 
+    patch('scipy-clang.patch', when='@1.5.0: %clang')
```

Comment:
Has this patch been submitted upstream?

Reply:
No, I haven't tried.

Reply:
I think it would be worth submitting a PR for that to get their feedback. I'm sure other users have encountered this problem.

Reply:
https://github.com/scipy/scipy/pull/14092

Reply:
This was merged! 🚀

Reply:
Since this was merged, I think we can do:
```suggestion
    patch('scipy-clang.patch', when='@1.5.0 %clang')
```

Reply:
Should probably be `@1.5.0:1.6.3` since that's the latest release. I can add those new release if you'd like as well.

https://github.com/scipy/scipy/releases

Reply:
New releases are already added in develop.

Reply:
Oh cool, I'll rebase and make the change then.

Reply:
I don't think a rebase is actually necessary but you can if you want.

Reply:
Done.